src/nodes.coffee

```diff
@@ -1223,18 +1223,128 @@ exports.Class = class Class extends Base
     klass = new Assign @variable, klass if @variable
     klass.compileToFragments o
 
+#### Import and Export
+
+exports.Module = class Module extends Base
+  constructor: (@type, @clause, @moduleName, @default = no) ->
+    if @type isnt 'import' and @type isnt 'export'
+      @error 'module type must be import or export'
+
+    if @moduleName? and @moduleName instanceof StringWithInterpolations
+      @error 'the name of the module to be imported from must be an uninterpolated string'
```

Comment:
I noticed that this error becomes much nicer if you use `@moduleName.error` instead of `@error`:

```
$ ./bin/coffee -pe 'import a from "a#{}"'
[stdin]:1:15: error: the name of the module to be imported from must be an uninterpolated string
import a from "a#{}"
              ^^^^^^
```